AztecTests/HTML/ElementNodeTests.swift

```diff
@@ -1374,4 +1374,18 @@ class ElementNodeTests: XCTestCase {
         }
 
     }
+
+    // MARK: - Bug fixes
+    // https://github.com/wordpress-mobile/WordPress-Aztec-iOS/issues/90
+    func testInsertNewlineAfterDivShouldNotCrash() {
```

Comment:
Could you document the test similarly to how other tests are documented?

It's a tool we're using to improve test maintenance and readability.  It also describes the logic so we can compare it against the actual test implementation.